Rakefile

```diff
@@ -3,7 +3,7 @@ require "rake/testtask"
 
 name = "stringio"
 
-if RUBY_PLATFORM =~ /java/
+if RUBY_ENGINE == "jruby"
```

Comment:
Could you use `case RUBY_ENGINE`/`when "jruby"`/`"ruby"` because all `if`/`elsif` use `RUBY_ENGINE ==`?